v5/owner.apib

```diff
@@ -0,0 +1,449 @@
+    + description_id: `00cd7cfd-e42d-11e2-8bf1-8a83f3373875` (string, required) - the description id
+
++ Request
+    + Headers
+
+            Authorization: Basic MDEyMzQ1Njc4YWJjZGVmOnBhcmt1
+
++ Response 200
+    + Attributes (Description)
+
++ Response 404
+Returned if parking space or description was not found, if the parking space doesn't belong to the user or description doesn't belong to the parking space.
+    + Body
+
++ Response 422
+Returned if an error occurred.
+    + Attributes (Api Error)
+
+### List images [GET /owner/parking-spaces/{parking_space_id}/images]
+Returns all images for one parking space
+
++ Parameters
+    + parking_space_id: `00cd7cfd-e42d-11e2-8bf1-8a83f3373875` (string, required) - the parking space id
+
++ Request
+    + Headers
+
+            Authorization: Basic MDEyMzQ1Njc4YWJjZGVmOnBhcmt1
+
++ Response 200 (application/json)
+     + Attributes (Image)
+
++ Response 404
+Returned if parking space was not found or it doesn't belong to the user.
+    + Body
+
+### Create a new Image [POST /owner/parking-spaces/{parking_space_id}/images]
++ Request (application/json)
+    + Attributes
+        + `content`: `w6JQTkcNChoNCgAAAA0KSUhEUgAAAMK3AAAAwrcIAwA...` (string, required) - base64 encoded image content
+
+    + Headers
+
+            Authorization: Basic MDEyMzQ1Njc4YWJjZGVmOnBhcmt1
+
++ Response 201 (application/json)
+    + Attributes (Image)
+
++ Response 404
+Returned if parking space was not found or it doesn't belong to the user.
+    + Body
+
++ Response 422
+Returned if an error occurred.
+    + Attributes (Api Error)
+
+### Delete an Image [DELETE /owner/parking-spaces/{parking_space_id}/images/{image_id}]
+
++ Parameters
+    + parking_space_id: `00cd7cfd-e42d-11e2-8bf1-8a83f3373875` (string, required) - the parking space id
+    + image_id: `00cd7cfd-e42d-11e2-8bf1-8a83f3373875` (string, required) - the image id
+
++ Request
+    + Headers
+
+            Authorization: Basic MDEyMzQ1Njc4YWJjZGVmOnBhcmt1
+
++ Response 200
+    + Attributes (Image)
+
++ Response 404
+Returned if parking space or image was not found or if the parking space doesn't belong to the user.
+    + Body
+
++ Response 422
+Returned if an error occurred.
+    + Attributes (Api Error)
+
+### List periods [GET /owner/parking-spaces/{parking_space_id}/periods]
+Returns all periods for one parking space
+
++ Parameters
+    + parking_space_id: `00cd7cfd-e42d-11e2-8bf1-8a83f3373875` (string, required) - the parking space id
+
++ Request
+    + Headers
+
+            Authorization: Basic MDEyMzQ1Njc4YWJjZGVmOnBhcmt1
+
++ Response 200 (application/json)
+     + Attributes (Period)
+
++ Response 404
+Returned if parking space was not found or it doesn't belong to the user.
+    + Body
+
+### Create a new Period [POST /owner/parking-spaces/{parking_space_id}/periods]
++ Request (application/json)
+    + Attributes
+        + `date_start`: `2017-12-15` (string) - can be null
```

Comment:
add nullable to the type and explain what null means in the description, also for `date_end`

Reply:
Done